springfield/firefox/templates/firefox/includes/download-button-thanks.html

```diff
@@ -18,10 +19,17 @@
     {% else %}
       {{ ftl('download-button-download-firefox') }}
     {% endif %}
-    <span class="mzp-c-button-icon-end">
-      <!-- TODO: replace with new protocol-assets version -->
-      <img src="{{ static('img/icons/m24-small/download-white.svg') }}" alt="" />
-    </span>
+    {% if flare_styles %}
+      <include:icon
+        icon_name="arrow-down"
+        hidden=true
+        extra_class="fl-icon-right" />
+    {% else %}
+      <span class="mzp-c-button-icon-end">
```

Comment:
The `mzp-c-` prefix is from Mozilla Protocol, and this shouldn't have any Protocol dependencies.